src/molecules/CheckboxGroup/CheckboxGroup.jsx

```diff
@@ -19,21 +19,24 @@ const CheckboxGroup = ({
 			newArr.push(sel);
 		}
 		updateSelected(newArr);
+		event.preventDefault();
```

Comment:
Eslint: [no-restricted-globals](https://eslint.org/docs/rules/no-restricted-globals) . Could we do this in another way?